dist/images/Makefile

```diff
@@ -13,36 +13,32 @@ all: ubuntu centos fedora
 SLASH = -
 ARCH = $(subst aarch64,arm64,$(subst x86_64,amd64,$(patsubst i%86,386,$(shell uname -m))))
 IMAGE_ARCH = $(SLASH)$(ARCH)
-DOCKERFILE_ARCH =
-ifeq ($(ARCH),arm64)
-        DOCKERFILE_ARCH=.arm64
-endif
 KERNEL_VERSION ?= `uname -r`
 OVS_BRANCH ?= master
 OVN_BRANCH ?= main
 OCI_BIN ?= docker
 
-# The image of ovnkube/ovn-daemonset-u should be multi-arched before using it on arm64
-ubuntu: bld
-	${OCI_BIN} build -t ovn-kube-u$(IMAGE_ARCH) -f Dockerfile.ubuntu$(DOCKERFILE_ARCH) .
 ifeq ($(ARCH),amd64)
-	${OCI_BIN} tag "ovn-kube-u$(IMAGE_ARCH):latest" \
-              "ovn-kube-u:latest"
+	IMAGE_ARCH=""
```

Comment:
I might be wrong here as well, why not $(uname -m) for double guarantee?

Reply:
This is to keep existing image naming for amd64 images. so if you build amd64 server we don't add the IMAGE_ARCH to the image name. This is for consistence with how currently images are named in ovn-kubernetes